README.md

```diff
@@ -56,6 +56,14 @@ To automatically set that, the above example uses the context variable `${{ gith
 
 - `platforms` (list, optional) List of target platform for build.
 
+- `target` (string, optional) the name of a build stage in the Dockerfile to target for the image. This allows multiple images
+  built from a single Dockerfile, e.g., "runtime-A" and "runtime-B".
+
+- `build-args` (list, optional) A list of build-arguments as newline-delimited `arg=value` string pairs. These may be
+  specified in the Dockerfile as `ARG` statements.
+
+- `additional-tags` (list, optional) A newline-delimited list of additional tags to be added to the built image.
```

Comment:
I think this needs more documentation pointing to docker/metadata-action, doesn't it? It looks like these tags have to be in some special format with a `type` marker.

Reply:
Not for the `type=raw` tags, you can just put string literals in there and the action will treat them correctly.

https://github.com/docker/metadata-action?tab=readme-ov-file#typeraw